code/game/objects/structures/lattice.dm

```diff
@@ -10,14 +10,13 @@
 	max_integrity = 50
 	layer = LATTICE_LAYER //under pipes
 	var/obj/item/stack/rods/stored
-	canSmoothWith = list(/obj/structure/lattice,
-	/turf/open/floor,
-	/turf/closed/wall,
-	/obj/structure/falsewall)
+	canSmoothWith = null
 	smooth = SMOOTH_MORE
```

Comment:
you'd have to make it SMOOTH_TRUE and have subtypes of everything minus shuttle walsl(which are just titanium walls)